docs/transactions/loyalty-program.md

```diff
@@ -0,0 +1,73 @@
+# Loyalty Program
+
+Loyalty Program is used to reward customers with points for their purchases, which can then be redeemed on future purchases.
+
+## Enabling Loyalty Program
+
+To enable Loyalty Program you will have to first check the Enable Loyalty Program. To
```

Comment:
...To enable Loyalty Program, you will have to first check the Enable Loyalty Program in the General Settings.